sorc/emcsfc_snow2mdl.fd/grib_utils.f

```diff
@@ -421,58 +372,36 @@ subroutine grib2_check (kgds, igdstmplen)
 
  end subroutine grib2_check
 
+!> Initialize grib2 arrays required by the ncep g2 library according to
```

Comment:
I am getting doxygen warnings on this routine as well.  Only the first line should have '!>', correct?

Reply:
Fixed